### ISSUE TITLE
[BUG] WAR

### ISSUE THREAD

**Issue by kedavlar** [guilds-plugin/Guilds#500]
## Description
one guild accepted the war with another guild, then they used the / g join command to enter the war, after which one member crashed and backed off after the timer expired. Now both guilds can't make wars anymore because it says `` war on hold, wait for it to end ''

## Expected Behavior
.

## Actual Behavior
cant do war 

## Possible Fix
.

## Steps to Reproduce
to replay, send a challenge to a guild, use the / g join command to join, exit the server and re-enter only after time runs out

## Context
.

## Your Environment
* Guilds version 3.5.6.2 DEV
* MC Version 1.16.5 tuinity

Sorry, i know its not the same thing, but i also get this error
https://pastebin.com/zSFTnfDL

**Comment by darbyjack** [comment]
Can this still be reproduced on the latest version?

**Comment by kedavlar** [comment]
For now i remove the war system from my server for the incompatibility with custom damage
